src/test/java/org/wso2/carbon/module/csv/util/parser/ParserTest.java

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.csv.util.parser;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.wso2.carbon.module.csv.util.parser.exception.ParserException;
+import org.wso2.carbon.module.csv.util.parser.model.Token;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+class ParserTest {
+
+    @Test
+    void testParse_singleValue_correctRangeShouldReturn() {
+
+        Tokenizer tokenizer = new Tokenizer();
```

Comment:
Shall we reduce the extra spacing to improve readability?

Reply:
removed unwanted spacing.